app/static/app/js/components/Map.jsx

```diff
@@ -263,6 +263,8 @@ https://a.tile.openstreetmap.org/{z}/{x}/{y}.png
       selectedOverlays: [],
       baseLayers: this.basemaps
     }).addTo(this.map);
+    
+    this.map.layerControl = this.autolayers;
```

Comment:
This is so that the plugin can add itself to the layer control. Probably can be done better, but I'm not sure how

Reply:
Would a hook help? https://github.com/OpenDroneMap/WebODM/blob/master/app/static/app/js/classes/plugins/Map.js#L15 https://github.com/OpenDroneMap/WebODM/blob/master/app/static/app/js/components/Map.jsx#L209

Reply:
Done!

Thanks for pointing me in the right directon :smile: